server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -126,7 +124,7 @@ class AddressIndexRepository @Inject()(
             matchQuery(
               field = "lpi.paoStartNumber",
               value = token
-            )),
+            ).boost(queryParams.buildingNumberLpiBoost)),
           tokensMap.get(Tokens.locality).map(token =>
```

Comment:
Do we test Tokens.locality against PAF.DEPENDENT_LOCALITY? That should be included if not...